narwhal/primary/src/synchronizer.rs

```diff
@@ -46,6 +46,10 @@ use crate::{aggregators::CertificatesAggregator, metrics::PrimaryMetrics, CHANNE
 #[path = "tests/synchronizer_tests.rs"]
 pub mod synchronizer_tests;
 
+/// Only try to accept or suspend a certificate, if it is within this limit above the
+/// locally highest processed round.
+const NEW_CERTIFICATE_ROUND_LIMIT: Round = 100;
```

Comment:
why is this different than the GC and how was this chosen?

Reply:
Feel free to suggest better names. `NEW_CERTIFICATE_ROUND_LIMIT` is the number of rounds above the highest round seen by this primary, where certificates that cannot be accepted are kept in memory. GC is the number of rounds below the highest committed round. So I think using the same value would be more confusing. It is chosen to tolerate a temporary loss of connection from the primary, e.g. currently it takes > 1 min to missing 100 rounds.